NomadUI/Core/NUIScrollbar.cpp

```
@@ -136,31 +162,101 @@ bool NUIScrollbar::onMouseEvent(const NUIMouseEvent& event)
         // Handle thumb dragging (mouse move events have button = None)
         // Use relative drag delta to preserve initial grab offset within thumb
         NUIRect trackRect = getTrackRect();
-        double deltaPixels, trackLength;
+        NUIRect thumbRect = getThumbRect();
+        double deltaPixels, trackLength, thumbLengthPixels;
         
         if (orientation_ == Orientation::Vertical)
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)